### PR TITLE
remove crashlythics and fix npe startup crash

### PR DESCRIPTION
**Pull request by shareefalis**

### CODE REVIEW

File: Tools/prepare_crashlytics.sh [hunk 1]
```diff
@@ -1,15 +0,0 @@
-#!/bin/bash
```
Comment (ianblenke) [inline Tools/prepare_crashlytics.sh:1]: You can't remove Tools/prepare_crashlytics.sh without removing it from .travis.yml

File: build.gradle [hunk 2]
```diff
@@ -1,36 +1,17 @@
-task wrapper(type: Wrapper) {
-    gradleVersion = '2.6'
-}
```
Comment (ianblenke) [inline build.gradle:3]: Why did you remove my wrapper?

File: build.gradle [hunk 3]
```diff
@@ -1,36 +1,17 @@
...
 buildscript {
     repositories {
         mavenCentral()
-        jcenter()
-        maven { url 'https://maven.fabric.io/public' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.1'
-        classpath 'io.fabric.tools:gradle:1.+'
+        classpath 'com.android.tools.build:gradle:1.3.0'
```
Comment (ianblenke) [inline build.gradle:6]: Should this be 1.3.0? It was 1.3.1

File: build.gradle [hunk 4]
```diff
@@ -60,7 +41,9 @@ android {
         debug.setRoot('build-types/debug')
         release.setRoot('build-types/release')
     }
-    lintOptions {
-        abortOnError false
+    defaultConfig {
+        targetSdkVersion 19
+    }
```
Comment (ianblenke) [inline build.gradle:46]: I swear there was a reason I had to targetSdkVersion 19...

File: src/org/linphone/AccountPreferencesFragment.java [hunk 5]
```diff
@@ -471,7 +471,7 @@ private void initializeTransportPreference(ListPreference pref) {
 		}
 		setListPreferenceValues(pref, entries, values);
 		
-		if (! isNewAccount) {
+		if (! isNewAccount && entries.indexOf(mPrefs.getAccountTransportString(n)) !=-1) {
```
Comment (ianblenke) [inline src/org/linphone/AccountPreferencesFragment.java:474]: This seems like a UDP transport fix? Should that be a separate PR?

File: src/org/linphone/LinphoneActivity.java [hunk 6]
```diff
@@ -358,12 +358,14 @@ public static void setBackgroundColorTheme(Context context){
...
-			topLayout.setBackgroundResource(R.drawable.background_theme_yellow);
-		}else{
+		if (topLayout != null){
+			if(background_color_theme.equals("Red")) {
+				topLayout.setBackgroundResource(R.drawable.background_theme_red);
+			}else if(background_color_theme.equals("Yellow")) {
+				topLayout.setBackgroundResource(R.drawable.background_theme_yellow);
+			}else{
 			topLayout.setBackgroundResource(R.drawable.background);
+			}
```
Comment (ianblenke) [inline src/org/linphone/LinphoneActivity.java:368]: This seems like a crash fix, not a Crashlytics fix. Should this be a separate PR?

File: build.gradle (hunk 2 above)
Comment (shareefalis) [inline build.gradle:3]: https://github.com/VTCSecureLLC/ace-android/commit/7ba298789fd51e72ffb1f8c319ad2a2c9a2eb5db 
It was part of the  crashlythicsndk commit

File: src/org/linphone/LinphoneActivity.java (hunk 6 above)
Comment (shareefalis) [inline src/org/linphone/LinphoneActivity.java:368]: Yeah i couldn't build without these. I could do a cherry-pick and force revert to the previous head.

File: build.gradle (hunk 4 above)
Comment (shareefalis) [inline build.gradle:46]: https://github.com/VTCSecureLLC/ace-android/commit/4d4f2ea27b640fc8042b5f9caf85e6e5b3e725ff
 look like you had it removed here. accidentally?
